packages/core/src/views/connect/Index.svelte

```diff
@@ -111,6 +111,9 @@
         chains: [{ namespace: 'evm', id: '0x1' }]
       }
 
+      if (instance)
+        selectedWallet.instance = instance
+
```

Comment:
```suggestion
```

We can remove this block and just add the instance to the selected wallet since it is optional in the types so it is fine if it is `undefined`:

```javascript
selectedWallet = {
  label,
  icon: loadedIcon,
  provider,
  instance,
  accounts: [],
  chains: [{ namespace: 'evm', id: '0x1' }]
}
```